tensornetwork/backends/jax/jitted_functions_test.py

```diff
@@ -54,7 +55,9 @@ def test_gmres_on_small_known_problem(dtype):
   def A_mv(x):
     return A @ x
   tol = A.size*jax.numpy.finfo(dtype).eps
-  x, _, _, _ = gmres.gmres_m(A_mv, [], b, x0, tol, tol, n_kry, maxiter)
+  precision = jax.lax.Precision.HIGHEST
```

Comment:
If this is to be hardcoded, perhaps once at the top of the file would be best?

Reply:
good point